usim_pytest/test_types/test_time.py

```diff
@@ -173,3 +173,13 @@ class TestTimeIteration:
             if iteration == 5:
                 break
             iteration += 1
+
+    @via_usim
+    async def test_interval_exceeded(self):
+        try:
+            async for _ in interval(20):
+                await (time + 40)
+        except IntervalExceeded:
+            assert True
+        else:
+            assert False
```

Comment:
Could you please also add a test for something that exactly takes ``period`` of time?

Reply:
Added this test now.